OWNERS

```diff
@@ -1,6 +1,6 @@
-approvers:
-- cm-maintainers
 reviewers:
-- cm-maintainers
-- spectralhiss
-- thatsmrtalbot
+  - TrilokGeer
```

Comment:
The owners and maintainers from upstream list should be retained.

Reply:
This can be maintained as similar to cert-manager downstream and possibly independent of carry PRs.

Reply:
Thank you! closing this PR.